client/src/Backend/GameLogic/GameManager.ts

```diff
@@ -4792,27 +4792,29 @@ class GameManager extends EventEmitter {
    * know the location of either planet. Takes into account links.
    */
   getDist(fromId: LocationId, toId: LocationId): number {
-    const from = this.entityStore.getPlanetWithId(fromId);
-    const to = this.entityStore.getPlanetWithId(toId);
-
-    if (!from) throw new Error('origin planet unknown');
-    if (!to) throw new Error('destination planet unknown');
-    if (!isLocatable(from)) throw new Error('origin location unknown');
-    if (!isLocatable(to)) throw new Error('destination location unknown');
-
-    const wormholeFactors = this.getWormholeFactors(from, to);
-
-    let distance = this.getDistCoords(from.location.coords, to.location.coords);
+    const planetFrom = this.entityStore.getPlanetWithId(fromId);
+    if (!isLocatable(planetFrom)) {
+      throw new Error(`origin planet not locatable (fromId: ${fromId}) (locationId: ${planetFrom?.locationId})`)
+    }
 
-    if (wormholeFactors) {
-      distance /= wormholeFactors.distanceFactor;
+    const planetTo = this.entityStore.getPlanetWithId(toId);
+    if (!isLocatable(planetTo)) {
+      throw new Error(`origin planet not locatable (toId: ${fromId}) (locationId: ${planetTo?.locationId})`)
     }
 
-    return distance;
+    const wormholeFactors = this.getWormholeFactors(planetFrom, planetTo);
+    const distance = this.getDistCoords(planetFrom.location.coords, planetTo.location.coords);
+    const distanceFactor = wormholeFactors
+      ? wormholeFactors.distanceFactor
+      : 1;
+
+    return distance / distanceFactor;
   }
 
   /**
-   * Gets the distance between two coordinates in space.
+   * Gets the distance between two coordinates in space using manhattan geometry
+   *
+   * @see https://en.wikipedia.org/wiki/Taxicab_geometry
```

Comment:
Added information about what kind of method we are using to to get distance coords, not everyone is familiar with manhattan geometry aka taxicab distance calculation. Learned this a couple of years ago when doing Advent of Code 😂